beta/src/pages/learn/add-react-to-a-website.md

```diff
@@ -1,56 +1,56 @@
 ---
-title: Add React to a Website
+title: किसी वेबसाइट पर React जोड़ें
 ---
 
 <Intro>
 
-React has been designed from the start for gradual adoption, and you can use as little or as much React as you need. Whether you're working with micro-frontends, an existing system, or just giving React a try, you can start adding interactive React components to an HTML page with just a few lines of code—and no build tooling!
+React को शुरू से ही धीरे-धीरे अपनाने के लिए डिज़ाइन किया गया है, और आप जितना चाहें उतना कम या ज्यादा React का उपयोग कर सकते हैं। चाहे आप माइक्रो-फ्रंटेंड, एक मौजूदा सिस्टम के साथ काम कर रहे हों, या सिर्फ React को आज़मा रहे हों, आप HTML पेज पर इंटरेक्टिव React कौम्पोनॅन्ट को कोड की कुछ पंक्तियों के साथ जोड़ना शुरू कर सकते हैं - और कोई बिल्ड टूलिंग नहीं!
 
 </Intro>
 
-## Add React in one minute {/*add-react-in-one-minute*/}
+## एक मिनट में React जोड़ें {/*add-react-in-one-minute*/}
 
-You can add a React component to an existing HTML page in under a minute. Try this out with your own website or [an empty HTML file](https://gist.github.com/rachelnabors/7b33305bf33776354797a2e3c1445186/archive/859eac2f7079c9e1f0a6eb818a9684a464064d80.zip)—all you need is an internet connection and a text editor like Notepad (or VSCode—check out our guide on [how to set yours up](/learn/editor-setup/))!
+आप एक मिनट से भी कम समय में मौजूदा HTML पृष्ठ पर एक React कौम्पोनॅन्ट जोड़ सकते हैं। इसे अपनी वेबसाइट या [एक खाली HTML फ़ाइल](https://gist.github.com/rachelnabors/7b33305bf33776354797a2e3c1445186/archive/859eac2f7079c9e1f0a6eb818a9684a464064d80.zip) के साथ आज़माएं—आपको बस एक इंटरनेट कनेक्शन और नोटपैड (या VSCode—को [कैसे सेट अप करें](/learn/editor-setup/) इस पर हमारी मार्गदर्शिका देखें) जैसे टेक्स्ट एडिटर की आवश्यकता है!
 
-### Step 1: Add an element to the HTML {/*step-1-add-an-element-to-the-html*/}
+### चरण 1: HTML में एक एलिमेंट जोड़ें {/*step-1-add-an-element-to-the-html*/}
 
-In the HTML page you want to edit, add an HTML element like an empty `<div>` tag with a unique `id` to mark the spot where you want to display something with React.
+जिस HTML पृष्ठ में आप संपादित करना चाहते हैं, जहां आप React के साथ कुछ प्रदर्शित करना चाहते हैं, उस स्थान पर एक खाली `<div>` टैग के साथ एक अनोखा ‘id’ वाला HTML एलिमेंट जोड़ें।
```

Comment:
```
जिस HTML पृष्ठ में आप संपादित करना चाहते हैं, जहां आप React के साथ कुछ प्रदर्शित करना चाहते हैं, उस स्थान पर एक खाली `<div>` टैग के साथ एक अनोखा ‘id’ वाला HTML एलिमेंट जोड़ें।
``` 
=> 
```
जिस HTML पेज को एडिट करके आप React के साथ कुछ डिस्प्ले करना चाहते हैं, वहां पर  एक खाली `<div>` टैग के साथ एक यूनिक ‘id’ वाला HTML एलिमेंट ऐड करें।
```

I have restructured complete line, let me know if you have any other inputs on this